epub34/rs/index.html

```diff
@@ -1205,22 +1205,31 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
-							user interface reasons). If it does, it SHOULD preserve as much as possible the cascade model of CSS.
-							Reading system developers are expected to also publicly document the user agent style sheets and how they
-							interact with EPUB creator's style settings.</p>
-					</li>
-					<li>
-						<p id="confreq-css-user-styles">
-							It MAY provide the user with an affordance that reverts the display to the author's stylesheet.</p>
+						<p id="confreq-css-rs-html-default">SHOULD support the [[html]] <a data-cite="html#rendering"
+								>suggested default rendering</a> in their user agent style sheet(s).</p>
 					</li>
 				</ul>
 
-				<p id="confreq-css-rs-html-default">In addition to supporting CSS properties as defined above, a reading
-					system's user agent style sheet SHOULD support the [[html]] <a data-cite="html#rendering">suggested
-						default rendering</a>.</p>
+				<div class="note">
+					<p>Reading system developers are expected to publicly document their user agent style sheet(s) and
+						how they interact with EPUB creator's style settings. </p>
+				</div>
+
+				<section id="sec-css-overrides">
+					<h4>Author styling overrides</h4>
+
+					<p id="confreq-css-overrides">A reading system MAY override parts of an [=EPUB publication|EPUB
+						publication's] styling (e.g., for ergonomic or user interface reasons). Morevoer, they MAY allow
+						users to set default themes and style preferences (e.g., for more accessible reading) that
+						further alter the original design of a publication.</p>
 
-				<!-- <p>Reading system developers should implement CSS support at the level of major browsers.</p> -->
+					<p id="confreq-css-user-styles">These changes often make it impractical to render an EPUB
+						publication exactly as designed by an author, even if that is what the user wants. To assist
+						users in viewing an EPUB publication as closely as possible to the author intent, however,
+						reading systems MAY provide an affordance that reverts as much reading system-applied styling as
+						feasible. For example, it could reset any user-applied styling and themes back to the reading
+						system defaults.</p>
```

Comment:
```suggestion
						users in viewing an EPUB publication as closely as possible to the author intent, however,
						reading systems MAY provide an affordance that reverts as much reading system-applied styling as
						feasible. For example, it could reset any user-applied styling and themes back to the reading
						system defaults.</p>
```

- The term 'user', though precise, is misleading me; maybe 'reader' is better.
- The example is not the one I would use; what I am looking for to the author's defaults, and not the reading system defaults.

Reply:
> The term 'user', though precise, is misleading me; maybe 'reader' is better.

I’m not generally a fan of the word ‘user.’ People sometimes refer to their devices (i.e. a Kindle) as a ‘reader.’  In this case, ‘user’ is the less ambiguous term. Unless we want to add an adjective like “human reader.”

>I am looking for to the author's defaults, and not the reading system defaults.

Apple Books software has a theme called “Original” that will display some of the css in the epub package. 

1230-1231 Perhaps “author original settings” instead of “reading system defaults”

Reply:
> Apple Books software has a theme called “Original” that will display some of the css in the epub package.

I must admit I did not know that, but that is something that I am looking for (modulo what "some" means). 

I had a look with a very simple epub book which does, essentially, one thing with the content via CSS, namely it requires justification. That feature works: none of the other Apple themes justify by default (and they also change background color and font), and pushing that option switches back to the original background color and text justification.

(The book does not set the background color which seems to set to black, which probably corresponds to the fact that my system is set to be in dark mode.)